packages/api/src/services/repository-location/repository-location.mapper.ts

```diff
@@ -0,0 +1,13 @@
+import {RepositoryLocation} from '../../models/repository-location';
+
+export class RepositoryLocationMapper {
```

Comment:
Create a Mapper abstract or normal class that we should extend where we are going to have some reusable logic as well as binding the generic types

Reply:
Done